hexrdgui/llnl_import_tool_dialog.py

```diff
@@ -39,6 +39,65 @@
 from hexrdgui.utils.dialog import add_help_url
 
 
+class AtlasConfig:
```

Comment:
I think it'd be nicer if `AtlasConfig` was on its own separate file, such as `atlas_config.py`. But we can leave it for this PR.